docs/platform-support.md

```diff
@@ -0,0 +1,44 @@
+# Platform Support
+
+The OpenTelemetry Collector will be supported following a tiered platform support model to balance between the aim to support as many platforms as possible and to guarantee stability for the most important platforms. The platform support for the OpenTelemetry Collector is broken into three tiers with different levels of support for each tier  and aligns with the current test strategy.
+
+## Current Test Strategy
+
```

Comment:
I would make this less concrete as to what specific versions we use for testing, outlining instead how we decide these versions. For CI testing this has de facto been "the latest OS version supported on Github runners".

Reply:
Removed references to specific OS versions.